packages/cfd/src/Containers/mt5-trade-modal.tsx

```diff
@@ -55,6 +55,7 @@ const MT5TradeModal = observer(
                     platform={platform}
                     ctrader_tokens={ctrader_tokens}
                     dxtrade_tokens={dxtrade_tokens}
+                    ctraderTokenCall={loadCTraderTokens}
```

Comment:
This is being passed as props in so many places, cant we just call it in the component itself ?

Reply:
It is passed as props since it is used in config.tsx file and I do not want to use observer in that file. It is passed as props in 2 components, I'll pass it as prop in just 1 place.